### ISSUE TITLE
tdlib is 9 month old

### ISSUE THREAD

**Issue by weirdo-neutrino** [alexander-akhmetov/python-telegram#304]
If updating is not an option, it can be useful to print a warning, that a local version of library, that can be not up-to-date, is used.
https://github.com/alexander-akhmetov/python-telegram/blob/master/telegram/tdjson.py#L13

As an example there is no support for video stickers as they were included only half a year ago: https://t.me/telegram/168

**Comment by weirdo-neutrino** [comment]
```python
system_library = ctypes.util.find_library("libtdjson")
```

This will never find system wide library. It will look lib**libtdjson**.so. You need to change it to

```python
system_library = ctypes.util.find_library("tdjson")
```

https://docs.python.org/3/library/ctypes.html#finding-shared-libraries

**Comment by alexander-akhmetov** [comment]
Hi! Thanks for reporting and the PR, there is indeed a bug in find_library. 

Releasing python-telegram with a newer tdlib version isn't a problem, I will do that. Meanwhile, as a workaround you can also specify the path to the library directly: `Telegram(library_path='...')`

**Comment by alexander-akhmetov** [comment]
Released `0.16.0` with your change and tdlib `v1.8.5`. Thanks!
